guides/source/.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_gem:
+  rubocop-rails-omakase: rubocop.yml
+
+Markdown:
+  Autodetect: false
```

Comment:
Even though this same setting exists in root `.rubocop.yml`, without this there are 15 offenses detected.

For example, this snippet actually raises a warning when parsing the ruby:
https://github.com/rails/rails/blob/a7c8a20825be5ac8d96566ec5dbe9daa85568bc8/guides/source/action_controller_overview.md?plain=1#L122-L124

Reply:
You probably want a `inherit_from` in this file as well. That should solve this, and makes it possible to run rubocop from this working directory. Otherwise it will not have loaded the extensions and fail when encountering the unknown `Markdown` toplevel key.

Reply:
My thought was to not mix rules, because the guides are intended to be run in the users application, not the Rails framework -- where the only linter that should matter is omakase.